reference/latest/src/main/resources/assets/fabric-docs-reference/lang/en_us.json

```diff
@@ -1,4 +1,8 @@
 {
   "effect.fabric-docs-reference.tater": "Tater",
-  "item.minecraft.potion.effect.tater": "Tater Potion"
+  "item.minecraft.potion.effect.tater": "Tater Potion",
```

Comment:
Is it really best practice to name the key `item.minecraft.potion.effect.tater`? I've always figured that the `minecraft` bit is supposed to be reserved for vanilla, kind of like a namespace.

Reply:
Yes, but by design potion effect names go under the `minecraft` namespace.